Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
```But I also believe that, to the user new to Katana that has never used Plastic, reading code filled with .scalable(...), Value(scalable: ..., fixed: ...) would be quite puzzling. ```
You can't use a library without knowing the basic concepts. Value is a basic concept of Plastic. Because of this I don't buy the puzzling argument. I agree with the boilerplate and in fact **I agree with the layout DSL**. I just don't want to extend this DSL everywhere

```
"Plastic scales dimensions for you in order to make your views lay out consistently across different screen sizes. If you don't want something to be scaled, just use .fixed"
```
in plastic you have to use fixed also when things are scaled. I can give you infinite examples of this